.eslintrc.json

```diff
@@ -1,6 +1,13 @@
 {
     "extends": "airbnb",
      "rules": {
-        "no-console": 0
+        "no-console": 0,
+        "comma-dangle": ["error", {
```

Comment:
The original linter recommends trailing commas in arrow functions as well, which actually causes a syntax error (I think its only valid in ecma7, the backend however uses only ecma6. So I basically overrode the rule to make it so you DON'T do trailing commas in arrow functions. I kept the rest enabled tho cuz I do like the benefit of trailing commas in newline-separated lists and objects.